docker-app/qfieldcloud/subscription/models.py

```diff
@@ -794,7 +794,7 @@ def create_subscription(
         plan: Plan,
         created_by: Person,
         active_since: Optional[datetime] = None,
-    ) -> Tuple["Subscription", "Subscription"]:
+    ) -> tuple[Optional["AbstractSubscription"], "AbstractSubscription"]:
```

Comment:
Why don't you use the definition from the docs: `tuple[AbstractSubscription | None, AbstractSubscription]` ?

Reply:
Fixed.